pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -271,7 +271,11 @@ func (p *DefaultProvider) createLaunchTemplate(ctx context.Context, options *ami
 		TagSpecifications: []*ec2.TagSpecification{
 			{
 				ResourceType: aws.String(ec2.ResourceTypeLaunchTemplate),
-				Tags:         utils.MergeTags(options.Tags, map[string]string{v1beta1.TagManagedLaunchTemplate: options.ClusterName, v1beta1.LabelNodeClass: options.NodeClassName}),
+				Tags: utils.MergeTags(options.Tags, map[string]string{
+					v1beta1.EksClusterNameAnnotationKey: options.ClusterName,
```

Comment:
So this adds the eks cluster annotation tag on now, is this right?

Reply:
Correct, according to @mikestef9, it should be added to all resources Karpenter creates.